lib/twilio/call.ts

```diff
@@ -669,6 +669,10 @@ class Call extends EventEmitter {
         data: { audioConstraints },
       }, this);
 
+      if (this._options.onGetUserMedia) {
```

Comment:
What do you think about `onGetUserMediaResolved`?

Reply:
I think it's too verbose..
if you look at existing events, we don't really use past tense. For example, after .accept() is invoked, we emit accept event, no event for failures. Same for other things. Using the convention onEventName has an implicit meaning that an event happened and succeeded.

Reply:
Just capturing our slack discussion here for future reference. This is more of a JS convention. For example, onClick, and not onClicked. onMouseMove, onTouchStart etc. There are exceptions of course. If we need to capture different stages, then we need to change. For example, connecting, connected, reconnecting, reconnected.